packages/client/hmi-client/src/services/charts.ts

```diff
@@ -74,12 +77,24 @@ export const createForecastChart = (
 	const yaxis = structuredClone(xaxis);
 	yaxis.title = options.yAxisTitle;
 
+	const translationMap = options.translationMap;
+	let labelExpr = '';
+	if (translationMap) {
+		Object.keys(translationMap).forEach((key) => {
+			labelExpr += `datum.value === '${key}' ? '${translationMap[key]}' : `;
+		});
+		labelExpr += " 'other'";
+	}
```

Comment:
This is really more of an artifact from running pyciemss, but it seems like it would be useful to have a translation map for variables/columns.